tests/test_retry_handlers.py

```diff
@@ -152,3 +155,54 @@ def test_exception_callback():
     assert test_callback.run_after_exception.call_count == retries, "The exception callback count " \
                                                                     "does not equal the number of " \
                                                                     "retries"
+
+
+@pytest.mark.L1
```

Comment:
Thanks for using a mark of `L1` instead of `l1` :-)

Reply:
YES! monospace for everything, needs the 'L' ;)